demisto_sdk/commands/generate_unit_tests/generate_unit_tests.py

```diff
@@ -192,52 +195,53 @@ def solve(self, generator: UnitTestsGenerator) -> TestModule:
 def run_generate_unit_tests(**kwargs):
     global logger
     input_path = kwargs.get('input_path', '')
-    test_data_path = kwargs.get('test_data_path', '')
     output_dir = kwargs.get('output_dir', '')
     commands = arg_to_list(kwargs.get('commands', ''))
     commands_examples_path = kwargs.get('examples', '')
     insecure = kwargs.get('insecure', False)
     use_demisto = kwargs.get('use_demisto', False)
+    append = kwargs.get('append', False)
 
     click.echo("================= Running Unit Testing Generator ===================")
     # validate inputs
+    input_path_obj = Path(input_path)
     if not input_path:
         print_error(
             'To use the generate_unit_tests version of this command please include an `input` argument')
         return 1
 
-    if input_path and not os.path.isfile(input_path):
+    if input_path and not input_path_obj.is_file():
```

Comment:
is_file checks for that the path is a file, and that it exists. did you mean to check both, or just one of the conditions?